src/main/java/com/google/sps/taskqueue/push/BlobstoreTaskWorker.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.taskqueue.push;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import com.google.appengine.api.blobstore.BlobstoreFailureException;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/blobstoreWorker")
```

Comment:
Can't regular users send requests to this endpoint as well? That would let arbitrary users delete blobs, as long as they know the key.

Reply:
Yes the user can.